base_images/Dockerfile.bamboo

```diff
@@ -14,11 +15,22 @@ curl "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/linux_64b
 amazon-linux-extras install epel -y && \
 yum update -y
 
-# install Python 3, pip, CLI utilities, session manager and other tools
+# install pip, CLI utilities, session manager and other tools
 RUN \
-yum install -y jq gcc git make openssl wget zip unzip tar python3-devel nodejs yarn python3-pip awscli session-manager-plugin.rpm procps parallel && \
+yum install -y jq gcc git make openssl wget zip unzip tar nodejs yarn python3-pip awscli session-manager-plugin.rpm procps parallel && \
 pip3 install -q --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org pydoc-markdown bandit safety flake8 black isort
 
+
+# install Python 3.9
+RUN \
+yum install -y openssl-devel bzip2-devel libffi-devel && \
+wget https://www.python.org/ftp/python/${PYTHON_VERSION}/Python-${PYTHON_VERSION}.tgz && \
+tar xvf Python-${PYTHON_VERSION}.tgz && \
+cd Python-*/ && \
+./configure --enable-optimizations && \
```

Comment:
Add `--enable-loadable-sqlite-extensions --enable-ipv6` to the configuration options.

Reply:
added